### ISSUE TITLE
OSS cluster inventory

### ISSUE THREAD

**Issue by johnSchnake** [vmware-tanzu/sonobuoy#965]
**Describe the solution you'd like**
Create an OSS cluster inventory plugin which may (or may not) run by default in Sonobuoy. It would provide a means to get good high-level data about what makes up the cluster.

To some extent this information is in the sonobuoy queries but there are other things which may not be or may be hard to parse for users.

**Comment by johnSchnake** [comment]
Have a POC for this and just needs tweaks really. A pretty small plugin so we'd like to lead with something more meaningful for a new Sonobuoy plugin. Still going to be something we do eventually I'm sure.

**Comment by vladimirvivien** [comment]
Closing.  This plugin has been implemented here - https://github.com/vmware-tanzu/sonobuoy-plugins/tree/master/cluster-inventory
